### PR TITLE
SYNPY-858  & SYNPY-860 Add articles and favicon

### PR DESCRIPTION
**Pull request by kimyen**

### CODE REVIEW

**Comment by kimyen** [comment]
screencapture-file-users-kimyenladia-documents-github-synapsepythonclient-docs-build-html-views-html-2018-12-21-11_46_04
screencapture-file-users-kimyenladia-documents-github-synapsepythonclient-docs-build-html-views-html-2018-12-21-12_13_05

File: docs/Views.rst [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+    query_results = syn.tableQuery("select * from %s" % view['id'])
+    data = query_results.asDataFrame()
+
+Updating Annotations using View
+===============================
+
+To update ``class`` annotation for ``file2``, simply update the view::
+
+    data["class"] = ["V", "VI"]
```
Comment (zimingd) [inline docs/Views.rst]: Since this is in its own section. Users may jump directly here and not have context of what `data` is from the previous section.

**Comment by kimyen** [comment]
screencapture-file-users-kimyenladia-documents-github-synapsepythonclient-docs-build-html-upload-html-2019-01-07-14_57_35

File: docs/Upload.rst [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    # Upload a new version of raw_data.txt, IMPLICIT UPDATE EXAMPLE
+    # Assuming that there is a file created with:
+    syn.store(File('path/to/old/raw_data.txt', parentId='syn123456'))
+
+    # To create a new version of that file, make sure you store it with the exact same name
+    new_file = syn.store(File('path/to/new_version/raw_data.txt',  parentId='syn123456'))
+
+Updating Annotations or Provenance without Changing Versions
+============================================================
+Any change to a File will automatically update its version. If this isn’t the desired behavior, such as minor cahnges to the metadata, you can set forceVersion=False with the Python or R clients. For command line, the commands set-annotations and set-provenance will update the metadata without creating a new version. Adding/updating annotations and provenance in the web client will also not cause a version change.
```
Comment (zimingd) [inline docs/Upload.rst]: typo: "changes"

Comment (zimingd) [reply docs/Upload.rst]: Inline add in-line code block:
"For command line, the commands `set-annotations` and `set-provenance`"

File: docs/Upload.rst [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+Important: Because Provenance is tracked by version, set forceVersion=False for minor changes to avoid breaking Provenance.
```
Comment (zimingd) [inline docs/Upload.rst]: Bold text this. Maybe make it a different color if that's possible (red? orange?).

**Pull request merged**